individual/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+import core
+from core.models import HistoryBusinessModel
+
+
+class Individual(HistoryBusinessModel):
+    first_name = models.CharField(db_column='FirstName', max_length=255, null=False)
+    last_name = models.CharField(db_column='LastName', max_length=255, null=False)
+    dob = core.fields.DateField(db_column='Dob', null=False)
+
+    class Meta:
+        managed = True
```

Comment:
I think `managed = True` is a default and it can be removed from Meta

Reply:
I think in case when we handle both managed and not managed tables explicitly specifying that will help with readability.

Reply:
Good point